gpii/node_modules/flowManager/src/CloudBasedFlowManager.js

```diff
@@ -44,6 +46,9 @@ fluid.defaults("gpii.flowManager.cloudBased", {
             defaultGradeNames: "gpii.flowManager.cloudBased.cloudStatus"
         }
     },
+    // File containing the full sha256 of the revision of the repository for
```

Comment:
The impl is fine - please explain further in this comment how this file is expected to get there.

Reply:
Done.

Reply:
I meant, how the file gets there, rather than how the path of the file gets here

Reply:
Done.  Hopefully I didn't overdo it.